### PR TITLE
Add scene_template SQL output

### PR DESCRIPTION
**Pull request by Traesh**
Related to https://github.com/TrinityCore/TrinityCore/commit/d4887311e39527e036419ebd966f399fdbdae67a

Based on Funjoker PR : https://github.com/TrinityCore/WowPacketParser/pull/260/files

### CODE REVIEW

File: WowPacketParser/Store/Storage.cs [hunk 1]
```diff
@@ -89,6 +89,9 @@ public static class Storage
 
         public static readonly DataBag<HotfixData> HotfixDatas = new DataBag<HotfixData>(new List<SQLOutput> { SQLOutput.hotfix_data });
 
+        // Scenes
+        public static readonly DataBag<SceneTemplate> Scenes = new DataBag<SceneTemplate>(new List<SQLOutput> { SQLOutput.scene_template });
```
Comment (Vincent-Michael) [inline WowPacketParser/Store/Storage.cs]: missing call to Storage.ClearContainers()

File: WowPacketParser/Store/Objects/SceneTemplate.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace WowPacketParser.Store.Objects
+{
+    [DBTableName("scene_template")]
+    public sealed class SceneTemplate : IDataModel
+    {
+        [DBFieldName("SceneId", true)]
+        public uint? SceneID;
+
+            [DBFieldName("Flags")]
```
Comment (Vincent-Michael) [inline WowPacketParser/Store/Objects/SceneTemplate.cs]: fix spaces

**Comment by funjoker** [comment]
Merge an i fix the spaces.
Seems like Traesh is kinda inactive atm

File: WowPacketParser/Store/Storage.cs (hunk 1 above)
Comment (Traesh) [inline WowPacketParser/Store/Storage.cs]: Done

**Comment by funjoker** [comment]
what about verified build column?

**Pull request merged**